tests/vshn.yml

```diff
@@ -38,6 +38,7 @@ parameters:
     services:
       vshn:
         enabled: true
+        externalDatabaseConnectionsEnabled: true
```

Comment:
Can you add this parameter to the default as well? But set it to false, as we can only use it on specific clusters.

Otherwise the compilation will probably fail for the remaining clusters, as there is no field with that name available.